frontend/simple/js/distribution/mincome-proportional.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+function incomeDistribution (incomes, minCome) {
+  let membersBelow = []
+  let belowMincomeTotalAmount = 0
+  let membersAbove = []
+  let aboveMincomeTotalAmount = 0
+
+  incomes.map(function (member) {
+    let belowMincome = minCome - member.amount
+    if (belowMincome > 0) {
+      membersBelow.push(member.name)
+      belowMincomeTotalAmount += belowMincome
+    }
+    let aboveMincome = member.amount - minCome
+    if (aboveMincome > 0) {
+      membersAbove.push(member.name)
+      aboveMincomeTotalAmount += aboveMincome
+    }
+  })
+
+  let totalProportionalDistribution = belowMincomeTotalAmount / aboveMincomeTotalAmount
+
+  let payments = []
+  incomes.map(function (memberFrom) {
+    if (membersAbove.find(function (memberAboveName) {
+      return memberFrom.name === memberAboveName
+    })) {
+      let aboveAmount = memberFrom.amount - minCome
+      let abovePercentage = aboveAmount / aboveMincomeTotalAmount
+      let distributionPercentage = abovePercentage * totalProportionalDistribution
+      let distributionAmount = distributionPercentage * aboveMincomeTotalAmount
```

Comment:
So this can be simplified I believe? Note how you use `aboveMincomeTotalAmount` twice, once in the numerator (line 32) and once in the denominator (line 30).

I think to calculate `distributionAmount` you just need to do: `distributionAmount  = totalProportionalDistribution * aboveAmount`